.gitignore

```diff
@@ -0,0 +1 @@
+.*.sw*
```

Comment:
what is this for ?

Reply:
These are for ignoring the temporary swap files generated by vim editor.

Reply:
ok. I usually put all my vim swap files under a different directory :)